zora/package.json

```diff
@@ -23,11 +23,11 @@
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "import": "./dist/index.js",
+      "import": "./es.js",
       "require": "./dist/index.cjs"
     },
     "./cjs": "./dist/index.cjs",
-    "./es": "./dist/index.js"
+    "./es": "./es.js"
```

Comment:
so I would keep this one pointing a ES built version while the "."."import" points to the zora/es.js you have added